File: Modul6/Soal3/PRAK603-2410817120013-OktaviaLailaQodari.py
n1, n2 = map(int, input().split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    baris1 = list(map(int, input().split()))
    baris2 = list(map(int, input().split()))
    hasil = [baris1[i] * baris2[i] for i in range(n1)]
    print(" ".join(map(str, hasil)))

File: Modul6/Soal3/PRAK603-2410817120013-OktaviaLailaQodari.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);
    
    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
        return 0;
    }
    
    int baris1[n1], baris2[n2];
    
    for (int i = 0; i < n1; i++) {
        scanf("%d", &baris1[i]);
    }
    
    for (int i = 0; i < n2; i++) {
        scanf("%d", &baris2[i]);
    }
    
    for (int i = 0; i < n1; i++) {
        printf("%d ", baris1[i] * baris2[i]);
    }
    printf("\n");
    
    return 0;
}